2019-03-29 11:03:09.032 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: 83367921c8994359aa973a05cf4180b6 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:09.071 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 83367921c8994359aa973a05cf4180b6 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:19.036 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: 06499e39076e4a569c60b8d848e0ab17 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:19.075 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 06499e39076e4a569c60b8d848e0ab17 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:28.301 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received message with unique_id: f2b1ba9f1c2d4baabd5879675939dafd __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:257
2019-03-29 11:03:28.308 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 9f9013bae56d4b2486b2f17825f1c822 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:28.347 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 9f9013bae56d4b2486b2f17825f1c822 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:28.348 11524 DEBUG oslo_concurrency.lockutils [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Acquired semaphore "refresh_cache-4ad84d2b-babc-4ef6-87eb-437ba51a77dc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2019-03-29 11:03:28.349 11524 DEBUG nova.network.neutronv2.api [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] _get_instance_nw_info() _get_instance_nw_info /usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py:1295
2019-03-29 11:03:28.351 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: e5e20b4b6bb34530a6e90144fb3a1f90 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:28.362 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: e5e20b4b6bb34530a6e90144fb3a1f90 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:28.467 11524 DEBUG neutronclient.v2_0.client [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?tenant_id=d4bbd972ca1b463ba028ccd1df983917&device_id=4ad84d2b-babc-4ef6-87eb-437ba51a77dc used request id req-fdf254a4-d65f-476c-ade1-8ab6720933f0 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:03:28.618 11524 DEBUG neutronclient.v2_0.client [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/networks.json?id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d used request id req-d7f638b9-d211-4322-89c3-11a5d871fc10 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:03:28.656 11524 DEBUG neutronclient.v2_0.client [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/floatingips.json?fixed_ip_address=10.10.10.3&port_id=7cf93e54-e1d8-4b03-a06b-7164797d1bcc used request id req-9f02e64c-c21f-4e0d-ab12-89e8904b9f78 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:03:28.759 11524 DEBUG neutronclient.v2_0.client [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/subnets.json?id=b9f5b815-a0ab-4476-a216-6b824fda2957 used request id req-37434480-4afd-4168-88be-ecde14d43f3a _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:03:28.854 11524 DEBUG neutronclient.v2_0.client [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?network_id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d&device_owner=network%3Adhcp used request id req-89dd461b-7f05-4e6c-aa16-4df0bf5ea314 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:03:28.855 11524 DEBUG nova.network.base_api [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Updating instance_info_cache with network_info: [{"profile": {}, "ovs_interfaceid": "7cf93e54-e1d8-4b03-a06b-7164797d1bcc", "preserve_on_delete": false, "network": {"bridge": "br-int", "subnets": [{"ips": [{"meta": {}, "version": 4, "type": "fixed", "floating_ips": [], "address": "10.10.10.3"}], "version": 4, "meta": {}, "dns": [], "routes": [], "cidr": "10.10.10.0/24", "gateway": {"meta": {}, "version": 4, "type": "gateway", "address": "10.10.10.1"}}], "meta": {"injected": false, "tenant_id": "d4bbd972ca1b463ba028ccd1df983917", "mtu": 1500}, "id": "968d1118-8921-4ef9-9cf6-534ff3d8bf7d", "label": "net10"}, "devname": "tap7cf93e54-e1", "vnic_type": "normal", "qbh_params": null, "meta": {}, "details": {"port_filter": true, "ovs_hybrid_plug": true}, "address": "fa:16:3e:73:f8:2c", "active": true, "type": "ovs", "id": "7cf93e54-e1d8-4b03-a06b-7164797d1bcc", "qbg_params": null}] update_instance_cache_with_nw_info /usr/lib/python2.7/site-packages/nova/network/base_api.py:48
2019-03-29 11:03:28.857 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: cc8d54274e264a6f8f433ff79758fa81 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:28.872 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: cc8d54274e264a6f8f433ff79758fa81 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:28.874 11524 DEBUG oslo_concurrency.lockutils [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Releasing semaphore "refresh_cache-4ad84d2b-babc-4ef6-87eb-437ba51a77dc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2019-03-29 11:03:28.876 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 4b61cae0108f47faa2bbada9f1509c9e exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:28.910 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 4b61cae0108f47faa2bbada9f1509c9e __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:28.917 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 4a3ee1b058e14115ad78d7d53752aea9 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:29.035 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 4a3ee1b058e14115ad78d7d53752aea9 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:29.039 11524 WARNING nova.image.glance [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No protocol specified in for api_server 'http://172.30.126.56:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2019-03-29 11:03:29.046 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 81f662ede9f349099f79b8c4007711f8 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:29.057 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 81f662ede9f349099f79b8c4007711f8 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:29.058 11524 DEBUG nova.virt.libvirt.driver [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Starting migrate_disk_and_power_off migrate_disk_and_power_off /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:7292
2019-03-29 11:03:29.061 11524 DEBUG nova.virt.libvirt.driver [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Shutting down instance from state 1 _clean_shutdown /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:2520
2019-03-29 11:03:29.092 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: 3f95ebd6c5fe4db6b223610a23b4da85 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:29.134 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 3f95ebd6c5fe4db6b223610a23b4da85 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:32.080 11524 INFO nova.virt.libvirt.driver [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Instance shutdown successfully after 3 seconds.
2019-03-29 11:03:32.087 11524 INFO nova.virt.libvirt.driver [-] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Instance destroyed successfully.
2019-03-29 11:03:32.090 11524 DEBUG nova.virt.libvirt.driver [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] skipping disk for instance-0000001a as it does not have a path _get_instance_disk_info /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:7073
2019-03-29 11:03:32.091 11524 DEBUG nova.virt.libvirt.driver [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] skipping disk for instance-0000001a as it does not have a path _get_instance_disk_info /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:7073
2019-03-29 11:03:32.092 11524 DEBUG oslo_concurrency.processutils [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Running cmd (subprocess): mv /var/lib/nova/instances/4ad84d2b-babc-4ef6-87eb-437ba51a77dc /var/lib/nova/instances/4ad84d2b-babc-4ef6-87eb-437ba51a77dc_resize execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:355
2019-03-29 11:03:32.105 11524 DEBUG oslo_concurrency.processutils [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CMD "mv /var/lib/nova/instances/4ad84d2b-babc-4ef6-87eb-437ba51a77dc /var/lib/nova/instances/4ad84d2b-babc-4ef6-87eb-437ba51a77dc_resize" returned: 0 in 0.013s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:385
2019-03-29 11:03:32.106 11524 DEBUG oslo_concurrency.processutils [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Running cmd (subprocess): mkdir -p /var/lib/nova/instances/4ad84d2b-babc-4ef6-87eb-437ba51a77dc execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:355
2019-03-29 11:03:32.114 11524 DEBUG oslo_concurrency.processutils [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CMD "mkdir -p /var/lib/nova/instances/4ad84d2b-babc-4ef6-87eb-437ba51a77dc" returned: 0 in 0.008s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:385
2019-03-29 11:03:32.115 11524 DEBUG os_brick.utils [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] ==> get_connector_properties: call u"{'execute': None, 'my_ip': '172.30.126.57', 'enforce_multipath': True, 'host': 'com57', 'root_helper': 'sudo nova-rootwrap /etc/nova/rootwrap.conf', 'multipath': False}" trace_logging_wrapper /usr/lib/python2.7/site-packages/os_brick/utils.py:146
2019-03-29 11:03:32.116 11524 INFO oslo.privsep.daemon [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Running privsep helper: ['sudo', 'nova-rootwrap', '/etc/nova/rootwrap.conf', 'privsep-helper', '--config-file', '/usr/share/nova/nova-dist.conf', '--config-file', '/etc/nova/nova.conf', '--privsep_context', 'os_brick.privileged.default', '--privsep_sock_path', '/tmp/tmp9D9M67/privsep.sock']
2019-03-29 11:03:32.750 11524 INFO oslo.privsep.daemon [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Spawned new privsep daemon via rootwrap
2019-03-29 11:03:32.751 11524 DEBUG oslo.privsep.daemon [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Accepted privsep connection to /tmp/tmp9D9M67/privsep.sock __init__ /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:313
2019-03-29 11:03:32.754 11524 INFO oslo.privsep.daemon [-] privsep daemon starting
2019-03-29 11:03:32.755 11524 INFO oslo.privsep.daemon [-] privsep process running with uid/gid: 0/0
2019-03-29 11:03:32.755 11524 INFO oslo.privsep.daemon [-] privsep process running with capabilities (eff/prm/inh): CAP_SYS_ADMIN/CAP_SYS_ADMIN/none
2019-03-29 11:03:32.755 11524 INFO oslo.privsep.daemon [-] privsep daemon running as pid 11939
2019-03-29 11:03:32.756 11524 DEBUG oslo.privsep.daemon [-] privsep: request[80071408]: (1,) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:03:32.756 11524 DEBUG oslo.privsep.daemon [-] privsep: reply[80071408]: (2,) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:03:32.761 11524 DEBUG oslo.privsep.daemon [-] privsep: request[80071408]: (3, 'os_brick.privileged.rootwrap.execute_root', ('cat', '/etc/iscsi/initiatorname.iscsi'), {}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:03:32.829 11524 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): cat /etc/iscsi/initiatorname.iscsi out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:03:32.832 11524 DEBUG oslo.privsep.daemon [-] CMD "cat /etc/iscsi/initiatorname.iscsi" returned: 0 in 0.004s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:03:32.832 11524 DEBUG oslo.privsep.daemon [-] privsep: reply[80071408]: (4, ('InitiatorName=iqn.1994-05.com.redhat:4d12e79a68be\n', '')) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:03:32.833 11524 DEBUG os_brick.initiator.linuxfc [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No Fibre Channel support detected on system. get_fc_hbas /usr/lib/python2.7/site-packages/os_brick/initiator/linuxfc.py:91
2019-03-29 11:03:32.833 11524 DEBUG os_brick.initiator.linuxfc [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No Fibre Channel support detected on system. get_fc_hbas /usr/lib/python2.7/site-packages/os_brick/initiator/linuxfc.py:91
2019-03-29 11:03:32.834 11524 DEBUG os_brick.utils [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] <== get_connector_properties: return (717ms) {'initiator': u'iqn.1994-05.com.redhat:4d12e79a68be', 'ip': u'172.30.126.57', 'platform': u'x86_64', 'host': u'com57', 'do_local_attach': False, 'os_type': u'linux2', 'multipath': False} trace_logging_wrapper /usr/lib/python2.7/site-packages/os_brick/utils.py:170
2019-03-29 11:03:32.836 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: cbe8e232de2748d1ba838ae8c17bd5be exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:32.913 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: cbe8e232de2748d1ba838ae8c17bd5be __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:32.922 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 4839b21f15eb4fd4bad9cc8b8f3e21e0 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:33.170 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 4839b21f15eb4fd4bad9cc8b8f3e21e0 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:33.175 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: bacc7c67ecb246c0a94fa2a663f883e0 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:33.190 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: bacc7c67ecb246c0a94fa2a663f883e0 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:33.197 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CAST unique_id: f3573e2e13524ba6bad0b2c9748477c5 exchange 'nova' topic 'compute.com58' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:33.199 11524 WARNING nova.image.glance [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No protocol specified in for api_server 'http://172.30.126.56:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2019-03-29 11:03:33.202 11524 DEBUG oslo_concurrency.lockutils [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" acquired by "nova.compute.manager._clear_events" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2019-03-29 11:03:33.203 11524 DEBUG oslo_concurrency.lockutils [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" released by "nova.compute.manager._clear_events" :: held 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2019-03-29 11:03:33.203 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-9db96fb6-0278-47d7-9826-148911a51fa1 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 77c0471d70224131bff0036f2d30b085 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:33.246 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 77c0471d70224131bff0036f2d30b085 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:33.962 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received message with unique_id: 230edd383e994b79bc8b12db4af768b7 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:257
2019-03-29 11:03:33.967 11524 DEBUG nova.compute.manager [req-23f51ebb-4bc0-427f-a43d-a430cced707b 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Received event network-vif-unplugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc external_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:6861
2019-03-29 11:03:33.968 11524 DEBUG oslo_concurrency.lockutils [req-23f51ebb-4bc0-427f-a43d-a430cced707b 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" acquired by "nova.compute.manager._pop_event" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2019-03-29 11:03:33.969 11524 DEBUG nova.compute.manager [req-23f51ebb-4bc0-427f-a43d-a430cced707b 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Unexpected attempt to pop events during shutdown _pop_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:350
2019-03-29 11:03:33.970 11524 DEBUG oslo_concurrency.lockutils [req-23f51ebb-4bc0-427f-a43d-a430cced707b 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" released by "nova.compute.manager._pop_event" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2019-03-29 11:03:33.971 11524 DEBUG nova.compute.manager [req-23f51ebb-4bc0-427f-a43d-a430cced707b 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] No waiting events found dispatching network-vif-unplugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc pop_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:366
2019-03-29 11:03:33.971 11524 WARNING nova.compute.manager [req-23f51ebb-4bc0-427f-a43d-a430cced707b 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Received unexpected event network-vif-unplugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc for instance
2019-03-29 11:03:35.584 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received message with unique_id: 824c0cf425834dc6a456f2180ade478c __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:257
2019-03-29 11:03:35.587 11524 DEBUG nova.compute.manager [req-af8c4607-2acb-45f2-a0ab-08e2f2e730d2 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Received event network-vif-plugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc external_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:6861
2019-03-29 11:03:35.588 11524 DEBUG oslo_concurrency.lockutils [req-af8c4607-2acb-45f2-a0ab-08e2f2e730d2 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" acquired by "nova.compute.manager._pop_event" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2019-03-29 11:03:35.588 11524 DEBUG nova.compute.manager [req-af8c4607-2acb-45f2-a0ab-08e2f2e730d2 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Unexpected attempt to pop events during shutdown _pop_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:350
2019-03-29 11:03:35.589 11524 DEBUG oslo_concurrency.lockutils [req-af8c4607-2acb-45f2-a0ab-08e2f2e730d2 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" released by "nova.compute.manager._pop_event" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2019-03-29 11:03:35.589 11524 DEBUG nova.compute.manager [req-af8c4607-2acb-45f2-a0ab-08e2f2e730d2 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] No waiting events found dispatching network-vif-plugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc pop_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:366
2019-03-29 11:03:35.590 11524 WARNING nova.compute.manager [req-af8c4607-2acb-45f2-a0ab-08e2f2e730d2 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Received unexpected event network-vif-plugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc for instance
2019-03-29 11:03:36.053 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received message with unique_id: a484c626a1154c6f808d341056925463 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:257
2019-03-29 11:03:36.057 11524 DEBUG nova.compute.manager [req-d64f896f-ca3f-44a2-8ff9-1e593a412760 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Received event network-changed external_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:6861
2019-03-29 11:03:36.058 11524 DEBUG oslo_concurrency.lockutils [req-d64f896f-ca3f-44a2-8ff9-1e593a412760 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Acquired semaphore "refresh_cache-4ad84d2b-babc-4ef6-87eb-437ba51a77dc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2019-03-29 11:03:36.059 11524 DEBUG nova.network.neutronv2.api [req-d64f896f-ca3f-44a2-8ff9-1e593a412760 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] _get_instance_nw_info() _get_instance_nw_info /usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py:1295
2019-03-29 11:03:36.061 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-d64f896f-ca3f-44a2-8ff9-1e593a412760 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] CALL msg_id: 2984edcf705f4554824f08d101dccd87 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:36.080 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 2984edcf705f4554824f08d101dccd87 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:36.196 11524 DEBUG neutronclient.v2_0.client [req-d64f896f-ca3f-44a2-8ff9-1e593a412760 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?tenant_id=d4bbd972ca1b463ba028ccd1df983917&device_id=4ad84d2b-babc-4ef6-87eb-437ba51a77dc used request id req-3f8862da-97c1-48c3-99ac-23f84af47f09 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:03:36.413 11524 DEBUG neutronclient.v2_0.client [req-d64f896f-ca3f-44a2-8ff9-1e593a412760 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/networks.json?id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d used request id req-9c7577b3-5d39-495e-ad35-af9b082f13d8 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:03:36.569 11524 DEBUG neutronclient.v2_0.client [req-d64f896f-ca3f-44a2-8ff9-1e593a412760 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/floatingips.json?fixed_ip_address=10.10.10.3&port_id=7cf93e54-e1d8-4b03-a06b-7164797d1bcc used request id req-e77538dc-bcf9-4750-9226-04ddde0c77a7 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:03:36.676 11524 DEBUG neutronclient.v2_0.client [req-d64f896f-ca3f-44a2-8ff9-1e593a412760 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/subnets.json?id=b9f5b815-a0ab-4476-a216-6b824fda2957 used request id req-394828f5-22ad-49b3-89bc-6ed67718c823 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:03:36.759 11524 DEBUG neutronclient.v2_0.client [req-d64f896f-ca3f-44a2-8ff9-1e593a412760 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?network_id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d&device_owner=network%3Adhcp used request id req-7de692a0-85ab-41d1-8730-2cf1cecd883e _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:03:36.760 11524 DEBUG nova.network.base_api [req-d64f896f-ca3f-44a2-8ff9-1e593a412760 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Updating instance_info_cache with network_info: [{"profile": {}, "ovs_interfaceid": "7cf93e54-e1d8-4b03-a06b-7164797d1bcc", "preserve_on_delete": false, "network": {"bridge": "br-int", "subnets": [{"ips": [{"meta": {}, "version": 4, "type": "fixed", "floating_ips": [], "address": "10.10.10.3"}], "version": 4, "meta": {}, "dns": [], "routes": [], "cidr": "10.10.10.0/24", "gateway": {"meta": {}, "version": 4, "type": "gateway", "address": "10.10.10.1"}}], "meta": {"injected": false, "tenant_id": "d4bbd972ca1b463ba028ccd1df983917", "mtu": 1500}, "id": "968d1118-8921-4ef9-9cf6-534ff3d8bf7d", "label": "net10"}, "devname": "tap7cf93e54-e1", "vnic_type": "normal", "qbh_params": null, "meta": {}, "details": {"port_filter": true, "ovs_hybrid_plug": true}, "address": "fa:16:3e:73:f8:2c", "active": false, "type": "ovs", "id": "7cf93e54-e1d8-4b03-a06b-7164797d1bcc", "qbg_params": null}] update_instance_cache_with_nw_info /usr/lib/python2.7/site-packages/nova/network/base_api.py:48
2019-03-29 11:03:36.762 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-d64f896f-ca3f-44a2-8ff9-1e593a412760 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] CALL msg_id: 29494bf52b004a7d8bf30b04a23bf140 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:36.783 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 29494bf52b004a7d8bf30b04a23bf140 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:36.784 11524 DEBUG oslo_concurrency.lockutils [req-d64f896f-ca3f-44a2-8ff9-1e593a412760 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Releasing semaphore "refresh_cache-4ad84d2b-babc-4ef6-87eb-437ba51a77dc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2019-03-29 11:03:38.738 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received message with unique_id: 6760afec009c42ad9e99c93314a208aa __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:257
2019-03-29 11:03:38.741 11524 DEBUG nova.compute.manager [req-c64fbdc9-5699-46af-b4e6-dcca62c10c5c 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Received event network-vif-plugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc external_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:6861
2019-03-29 11:03:38.741 11524 DEBUG oslo_concurrency.lockutils [req-c64fbdc9-5699-46af-b4e6-dcca62c10c5c 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" acquired by "nova.compute.manager._pop_event" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2019-03-29 11:03:38.742 11524 DEBUG nova.compute.manager [req-c64fbdc9-5699-46af-b4e6-dcca62c10c5c 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Unexpected attempt to pop events during shutdown _pop_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:350
2019-03-29 11:03:38.742 11524 DEBUG oslo_concurrency.lockutils [req-c64fbdc9-5699-46af-b4e6-dcca62c10c5c 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc-events" released by "nova.compute.manager._pop_event" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2019-03-29 11:03:38.742 11524 DEBUG nova.compute.manager [req-c64fbdc9-5699-46af-b4e6-dcca62c10c5c 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] No waiting events found dispatching network-vif-plugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc pop_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:366
2019-03-29 11:03:38.743 11524 WARNING nova.compute.manager [req-c64fbdc9-5699-46af-b4e6-dcca62c10c5c 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Received unexpected event network-vif-plugged-7cf93e54-e1d8-4b03-a06b-7164797d1bcc for instance
2019-03-29 11:03:39.046 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: 07f1b707015641c9a9edf318220915d1 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:39.087 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 07f1b707015641c9a9edf318220915d1 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:46.614 11524 DEBUG nova.virt.driver [-] Emitting event <LifecycleEvent: 1553828611.61, 4ad84d2b-babc-4ef6-87eb-437ba51a77dc => Stopped> emit_event /usr/lib/python2.7/site-packages/nova/virt/driver.py:1438
2019-03-29 11:03:46.614 11524 INFO nova.compute.manager [-] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] VM Stopped (Lifecycle Event)
2019-03-29 11:03:46.616 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-032491fc-2c48-4404-8a4c-15cee9ab9667 - - - - -] CALL msg_id: ac06bfea3c97412789a849a8c52bea48 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:46.664 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: ac06bfea3c97412789a849a8c52bea48 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:46.665 11524 DEBUG nova.compute.manager [req-032491fc-2c48-4404-8a4c-15cee9ab9667 - - - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Checking state _get_power_state /usr/lib/python2.7/site-packages/nova/compute/manager.py:1176
2019-03-29 11:03:46.667 11524 DEBUG nova.compute.manager [req-032491fc-2c48-4404-8a4c-15cee9ab9667 - - - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Synchronizing instance power state after lifecycle event "Stopped"; current vm_state: resized, current task_state: None, current DB power_state: 1, VM power_state: 4 handle_lifecycle_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:1088
2019-03-29 11:03:46.669 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-032491fc-2c48-4404-8a4c-15cee9ab9667 - - - - -] CALL msg_id: 324299aa7aee4609b425c5cd42b17c24 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:46.718 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 324299aa7aee4609b425c5cd42b17c24 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:46.719 11524 INFO nova.compute.manager [req-032491fc-2c48-4404-8a4c-15cee9ab9667 - - - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] During the sync_power process the instance has moved from host com58 to host com57
2019-03-29 11:03:49.050 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: abd6ce977ce04e6ca605303f3c709b51 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:49.089 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: abd6ce977ce04e6ca605303f3c709b51 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407






2019-03-29 11:03:55.559 11524 DEBUG oslo_service.periodic_task [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Running periodic task ComputeManager._poll_volume_usage run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 11:03:56.559 11524 DEBUG oslo_service.periodic_task [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Running periodic task ComputeManager._reclaim_queued_deletes run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 11:03:56.559 11524 DEBUG nova.compute.manager [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] CONF.reclaim_instance_interval <= 0, skipping... _reclaim_queued_deletes /usr/lib/python2.7/site-packages/nova/compute/manager.py:6556
2019-03-29 11:03:56.560 11524 DEBUG oslo_service.periodic_task [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Running periodic task ComputeManager._heal_instance_info_cache run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 11:03:56.560 11524 DEBUG nova.compute.manager [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Starting heal instance info cache _heal_instance_info_cache /usr/lib/python2.7/site-packages/nova/compute/manager.py:5884
2019-03-29 11:03:56.560 11524 DEBUG nova.compute.manager [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Rebuilding the list of instances to heal _heal_instance_info_cache /usr/lib/python2.7/site-packages/nova/compute/manager.py:5888
2019-03-29 11:03:56.562 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] CALL msg_id: 062d5e9e33cb42158a5cad61d6ce41ad exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:56.588 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 062d5e9e33cb42158a5cad61d6ce41ad __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:56.588 11524 DEBUG nova.compute.manager [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Didn't find any instances for network info cache update. _heal_instance_info_cache /usr/lib/python2.7/site-packages/nova/compute/manager.py:5960
2019-03-29 11:03:58.588 11524 DEBUG oslo_service.periodic_task [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Running periodic task ComputeManager._poll_rescued_instances run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 11:03:59.052 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: f3d2b6bfbb53428fb0468b45df762dbb exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:59.101 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: f3d2b6bfbb53428fb0468b45df762dbb __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:59.556 11524 DEBUG oslo_service.periodic_task [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Running periodic task ComputeManager._check_instance_build_time run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 11:03:59.559 11524 DEBUG oslo_service.periodic_task [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Running periodic task ComputeManager._poll_rebooting_instances run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 11:03:59.560 11524 DEBUG oslo_service.periodic_task [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Running periodic task ComputeManager.update_available_resource run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 11:03:59.562 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] CALL msg_id: 33b814f1faf0448e8872562024952311 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:59.581 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 33b814f1faf0448e8872562024952311 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:59.586 11524 DEBUG nova.compute.resource_tracker [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Auditing locally available compute resources for com57 (node: com57) update_available_resource /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:534
2019-03-29 11:03:59.627 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] CALL msg_id: 63ead550a92c47ce836aafec0cd7859a exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:59.656 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 63ead550a92c47ce836aafec0cd7859a __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:59.660 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] CALL msg_id: 1c9e653dc8ee4cf2ba5a3738cd3e91b7 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:59.672 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 1c9e653dc8ee4cf2ba5a3738cd3e91b7 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:03:59.674 11524 DEBUG nova.virt.libvirt.driver [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] skipping disk for instance-0000001a as it does not have a path _get_instance_disk_info /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:7073
2019-03-29 11:03:59.674 11524 DEBUG nova.virt.libvirt.driver [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] skipping disk for instance-0000001a as it does not have a path _get_instance_disk_info /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:7073
2019-03-29 11:03:59.722 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received message with unique_id: 154a151f11e54bf0b26965d586543f38 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:257
2019-03-29 11:03:59.727 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: b1176f9ab16b4217a490e3181bf64d51 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:03:59.773 11524 DEBUG nova.compute.resource_tracker [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Hypervisor: free VCPUs: 2 _report_hypervisor_resource_view /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:655
2019-03-29 11:03:59.773 11524 DEBUG nova.compute.resource_tracker [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Hypervisor/Node resource view: name=com57 free_ram=7105MB free_disk=290GB free_vcpus=2 pci_devices=[{"dev_id": "pci_0000_00_00_0", "product_id": "7190", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7190", "address": "0000:00:00.0"}, {"dev_id": "pci_0000_00_01_0", "product_id": "7191", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7191", "address": "0000:00:01.0"}, {"dev_id": "pci_0000_00_07_0", "product_id": "7110", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7110", "address": "0000:00:07.0"}, {"dev_id": "pci_0000_00_07_1", "product_id": "7111", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7111", "address": "0000:00:07.1"}, {"dev_id": "pci_0000_00_07_3", "product_id": "7113", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7113", "address": "0000:00:07.3"}, {"dev_id": "pci_0000_00_07_7", "product_id": "0740", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_0740", "address": "0000:00:07.7"}, {"dev_id": "pci_0000_00_0f_0", "product_id": "0405", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_0405", "address": "0000:00:0f.0"}, {"dev_id": "pci_0000_00_11_0", "product_id": "0790", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_0790", "address": "0000:00:11.0"}, {"dev_id": "pci_0000_00_15_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.0"}, {"dev_id": "pci_0000_03_00_0", "product_id": "07c0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07c0", "address": "0000:03:00.0"}, {"dev_id": "pci_0000_00_15_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.1"}, {"dev_id": "pci_0000_00_15_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.2"}, {"dev_id": "pci_0000_00_15_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.3"}, {"dev_id": "pci_0000_00_15_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.4"}, {"dev_id": "pci_0000_00_15_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.5"}, {"dev_id": "pci_0000_00_15_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.6"}, {"dev_id": "pci_0000_00_15_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.7"}, {"dev_id": "pci_0000_00_16_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.0"}, {"dev_id": "pci_0000_0b_00_0", "product_id": "07b0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07b0", "address": "0000:0b:00.0"}, {"dev_id": "pci_0000_00_16_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.1"}, {"dev_id": "pci_0000_00_16_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.2"}, {"dev_id": "pci_0000_00_16_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.3"}, {"dev_id": "pci_0000_00_16_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.4"}, {"dev_id": "pci_0000_00_16_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.5"}, {"dev_id": "pci_0000_00_16_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.6"}, {"dev_id": "pci_0000_00_16_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.7"}, {"dev_id": "pci_0000_00_17_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.0"}, {"dev_id": "pci_0000_13_00_0", "product_id": "07b0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07b0", "address": "0000:13:00.0"}, {"dev_id": "pci_0000_00_17_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.1"}, {"dev_id": "pci_0000_00_17_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.2"}, {"dev_id": "pci_0000_00_17_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.3"}, {"dev_id": "pci_0000_00_17_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.4"}, {"dev_id": "pci_0000_00_17_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.5"}, {"dev_id": "pci_0000_00_17_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.6"}, {"dev_id": "pci_0000_00_17_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.7"}, {"dev_id": "pci_0000_00_18_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.0"}, {"dev_id": "pci_0000_00_18_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.1"}, {"dev_id": "pci_0000_00_18_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.2"}, {"dev_id": "pci_0000_00_18_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.3"}, {"dev_id": "pci_0000_00_18_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.4"}, {"dev_id": "pci_0000_00_18_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.5"}, {"dev_id": "pci_0000_00_18_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.6"}, {"dev_id": "pci_0000_00_18_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.7"}] _report_hypervisor_resource_view /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:672
2019-03-29 11:03:59.774 11524 DEBUG oslo_concurrency.lockutils [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Lock "compute_resources" acquired by "nova.compute.resource_tracker._update_available_resource" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2019-03-29 11:03:59.775 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] CALL msg_id: 243dc5aae40d40acaf57161b0869d327 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:00.011 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: b1176f9ab16b4217a490e3181bf64d51 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:00.013 11524 DEBUG oslo_concurrency.lockutils [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc" acquired by "nova.compute.manager.do_confirm_resize" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2019-03-29 11:04:00.014 11524 DEBUG nova.compute.manager [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Going to confirm migration 58 do_confirm_resize /usr/lib/python2.7/site-packages/nova/compute/manager.py:3460
2019-03-29 11:04:00.015 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: b31e43ffe7c144ac9a099efd9735b210 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:00.024 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: b31e43ffe7c144ac9a099efd9735b210 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:00.027 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: d398d65fe2c048b994526fdfe9a3253b exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:00.075 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 243dc5aae40d40acaf57161b0869d327 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:00.077 11524 DEBUG nova.scheduler.client.report [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Refreshing aggregate associations for resource provider db732609-7867-486c-8e55-f52f70acfa62 _ensure_resource_provider /usr/lib/python2.7/site-packages/nova/scheduler/client/report.py:402
2019-03-29 11:04:00.119 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] CALL msg_id: 22458d234d8649f5b5d4d702f554e7c2 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:00.196 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: d398d65fe2c048b994526fdfe9a3253b __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:00.198 11524 WARNING nova.image.glance [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No protocol specified in for api_server 'http://172.30.126.56:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2019-03-29 11:04:00.201 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 8614e96ef19c496e9c0b3f2bf3ababf3 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:00.205 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 22458d234d8649f5b5d4d702f554e7c2 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:00.221 11524 DEBUG nova.scheduler.client.report [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Deleting stale allocation for instance 4ad84d2b-babc-4ef6-87eb-437ba51a77dc remove_deleted_instances /usr/lib/python2.7/site-packages/nova/scheduler/client/report.py:750
2019-03-29 11:04:00.256 11524 INFO nova.scheduler.client.report [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Deleted allocation for instance 4ad84d2b-babc-4ef6-87eb-437ba51a77dc
2019-03-29 11:04:00.257 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] CALL msg_id: dd5e4ecd5bd34a21ac008d7e2d342f42 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:00.323 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: dd5e4ecd5bd34a21ac008d7e2d342f42 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:00.325 11524 DEBUG nova.compute.resource_tracker [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Migration for instance 4ad84d2b-babc-4ef6-87eb-437ba51a77dc refers to another host's instance! _pair_instances_to_migrations /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:568
2019-03-29 11:04:00.326 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] CALL msg_id: 36d53de2e2e24d91b88c3f4b1694735f exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:00.352 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 8614e96ef19c496e9c0b3f2bf3ababf3 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:00.365 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 36d53de2e2e24d91b88c3f4b1694735f __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:00.368 11524 INFO nova.compute.resource_tracker [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Updating from migration 4ad84d2b-babc-4ef6-87eb-437ba51a77dc
2019-03-29 11:04:00.368 11524 DEBUG nova.objects.instance [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Lazy-loading 'flavor' on Instance uuid 4ad84d2b-babc-4ef6-87eb-437ba51a77dc obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1034
2019-03-29 11:04:00.370 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] CALL msg_id: ff4390d601b34b8b9988e53faee08731 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:00.418 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: ff4390d601b34b8b9988e53faee08731 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:00.421 11524 DEBUG nova.objects.instance [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Lazy-loading 'migration_context' on Instance uuid 4ad84d2b-babc-4ef6-87eb-437ba51a77dc obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1034
2019-03-29 11:04:00.422 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] CALL msg_id: f1e24d22cf514431890cc086f1f6eb30 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:00.425 11524 DEBUG neutronclient.v2_0.client [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?tenant_id=d4bbd972ca1b463ba028ccd1df983917&binding%3Ahost_id=com58&device_id=4ad84d2b-babc-4ef6-87eb-437ba51a77dc used request id req-13b42db4-ad19-4002-be5b-3f7bd66ea6d0 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:04:00.426 11524 DEBUG oslo_concurrency.lockutils [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Acquired semaphore "refresh_cache-4ad84d2b-babc-4ef6-87eb-437ba51a77dc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2019-03-29 11:04:00.426 11524 DEBUG nova.network.neutronv2.api [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] _get_instance_nw_info() _get_instance_nw_info /usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py:1295
2019-03-29 11:04:00.427 11524 DEBUG nova.objects.instance [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lazy-loading 'info_cache' on Instance uuid 4ad84d2b-babc-4ef6-87eb-437ba51a77dc obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1034
2019-03-29 11:04:00.428 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 6f83c238356f43c8bd8e0c2ecf7334af exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:00.436 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: f1e24d22cf514431890cc086f1f6eb30 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:00.439 11524 DEBUG nova.compute.resource_tracker [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Total usable vcpus: 2, total allocated vcpus: 1 _report_final_resource_view /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:688
2019-03-29 11:04:00.440 11524 INFO nova.compute.resource_tracker [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Final resource view: name=com57 phys_ram=8191MB used_ram=1024MB phys_disk=299GB used_disk=1GB total_vcpus=2 used_vcpus=1 pci_stats=[]
2019-03-29 11:04:00.445 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] CALL msg_id: ff8faac94e484a2a93bf5e2d6e699142 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:00.473 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 6f83c238356f43c8bd8e0c2ecf7334af __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:00.477 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: f730142102474809872347c0e96285b1 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:00.485 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: f730142102474809872347c0e96285b1 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:00.545 11524 DEBUG neutronclient.v2_0.client [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?tenant_id=d4bbd972ca1b463ba028ccd1df983917&device_id=4ad84d2b-babc-4ef6-87eb-437ba51a77dc used request id req-715f9bd0-0ffb-4758-baec-d0a47d292ab2 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:04:00.646 11524 DEBUG neutronclient.v2_0.client [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/networks.json?id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d used request id req-960e5f5f-d5ef-427d-87e7-c68fd9b90395 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:04:00.665 11524 DEBUG neutronclient.v2_0.client [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/floatingips.json?fixed_ip_address=10.10.10.3&port_id=7cf93e54-e1d8-4b03-a06b-7164797d1bcc used request id req-9b231125-98e0-416b-8465-ff041418c99e _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:04:00.723 11524 DEBUG neutronclient.v2_0.client [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/subnets.json?id=b9f5b815-a0ab-4476-a216-6b824fda2957 used request id req-ff8e73c3-0f2c-4a52-940e-a4b0ea1dbf5b _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:04:00.762 11524 DEBUG neutronclient.v2_0.client [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?network_id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d&device_owner=network%3Adhcp used request id req-47f32405-ce8c-4728-80a1-2241501418b0 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 11:04:00.764 11524 DEBUG nova.network.base_api [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: 4ad84d2b-babc-4ef6-87eb-437ba51a77dc] Updating instance_info_cache with network_info: [{"profile": {}, "ovs_interfaceid": "7cf93e54-e1d8-4b03-a06b-7164797d1bcc", "preserve_on_delete": false, "network": {"bridge": "br-int", "subnets": [{"ips": [{"meta": {}, "version": 4, "type": "fixed", "floating_ips": [], "address": "10.10.10.3"}], "version": 4, "meta": {}, "dns": [], "routes": [], "cidr": "10.10.10.0/24", "gateway": {"meta": {}, "version": 4, "type": "gateway", "address": "10.10.10.1"}}], "meta": {"injected": false, "tenant_id": "d4bbd972ca1b463ba028ccd1df983917", "mtu": 1500}, "id": "968d1118-8921-4ef9-9cf6-534ff3d8bf7d", "label": "net10"}, "devname": "tap7cf93e54-e1", "vnic_type": "normal", "qbh_params": null, "meta": {}, "details": {"port_filter": true, "ovs_hybrid_plug": true}, "address": "fa:16:3e:73:f8:2c", "active": true, "type": "ovs", "id": "7cf93e54-e1d8-4b03-a06b-7164797d1bcc", "qbg_params": null}] update_instance_cache_with_nw_info /usr/lib/python2.7/site-packages/nova/network/base_api.py:48
2019-03-29 11:04:00.766 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: e4609c40a1a44e0f9d995848eccba28e exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:00.968 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: e4609c40a1a44e0f9d995848eccba28e __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:00.969 11524 DEBUG oslo_concurrency.lockutils [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Releasing semaphore "refresh_cache-4ad84d2b-babc-4ef6-87eb-437ba51a77dc" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2019-03-29 11:04:00.970 11524 DEBUG oslo_concurrency.processutils [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Running cmd (subprocess): rm -rf /var/lib/nova/instances/4ad84d2b-babc-4ef6-87eb-437ba51a77dc_resize execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:355
2019-03-29 11:04:00.985 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: ff8faac94e484a2a93bf5e2d6e699142 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:00.986 11524 DEBUG oslo_concurrency.processutils [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CMD "rm -rf /var/lib/nova/instances/4ad84d2b-babc-4ef6-87eb-437ba51a77dc_resize" returned: 0 in 0.016s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:385
2019-03-29 11:04:00.988 11524 DEBUG nova.scheduler.client.report [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Refreshing aggregate associations for resource provider db732609-7867-486c-8e55-f52f70acfa62 _ensure_resource_provider /usr/lib/python2.7/site-packages/nova/scheduler/client/report.py:402
2019-03-29 11:04:03.281 11524 DEBUG nova.virt.libvirt.storage.rbd_utils [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] removing snapshot(nova-resize) on rbd image(4ad84d2b-babc-4ef6-87eb-437ba51a77dc_disk) remove_snap /usr/lib/python2.7/site-packages/nova/virt/libvirt/storage/rbd_utils.py:415
2019-03-29 11:04:03.294 11524 DEBUG nova.compute.resource_tracker [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Compute_service record updated for com57:com57 _update_available_resource /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:626
2019-03-29 11:04:03.295 11524 DEBUG oslo_concurrency.lockutils [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Lock "compute_resources" released by "nova.compute.resource_tracker._update_available_resource" :: held 3.521s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2019-03-29 11:04:03.295 11524 DEBUG oslo_service.periodic_task [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Running periodic task ComputeManager._poll_unconfirmed_resizes run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 11:04:03.977 11524 DEBUG nova.virt.libvirt.vif [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] vif_type=ovs instance=Instance(access_ip_v4=None,access_ip_v6=None,architecture='x86_64',auto_disk_config=True,availability_zone='nova',cell_name=None,cleaned=True,config_drive='True',created_at=2019-03-29T02:58:47Z,default_ephemeral_device=None,default_swap_device=None,deleted=False,deleted_at=None,device_metadata=<?>,disable_terminate=False,display_description='vm003',display_name='vm003',ec2_ids=<?>,ephemeral_gb=0,ephemeral_key_uuid=None,fault=<?>,flavor=Flavor(1),host='com58',hostname='vm003',id=26,image_ref='40f27f13-d552-4f7e-ae5d-2b85b126c85e',info_cache=InstanceInfoCache,instance_type_id=1,kernel_id='',key_data=None,key_name=None,keypairs=<?>,launch_index=0,launched_at=2019-03-29T03:03:37Z,launched_on='com58',locked=False,locked_by=None,memory_mb=512,metadata={},migration_context=<?>,new_flavor=None,node='com58',numa_topology=<?>,old_flavor=None,os_type=None,pci_devices=<?>,pci_requests=<?>,power_state=1,progress=0,project_id='d4bbd972ca1b463ba028ccd1df983917',ramdisk_id='',reservation_id='r-aj6jz56r',root_device_name='/dev/vda',root_gb=1,security_groups=<?>,services=<?>,shutdown_terminate=False,system_metadata={boot_roles='admin',clean_attempts='1',image_architecture='x86_64',image_base_image_ref='40f27f13-d552-4f7e-ae5d-2b85b126c85e',image_container_format='bare',image_disk_format='qcow2',image_min_disk='1',image_min_ram='0'},tags=<?>,task_state=None,terminated_at=None,updated_at=2019-03-29T03:03:38Z,user_data=None,user_id='9e9e53917dda421083fa6af425c6bfbf',uuid=4ad84d2b-babc-4ef6-87eb-437ba51a77dc,vcpu_model=<?>,vcpus=1,vm_mode=None,vm_state='resized') vif={"profile": {}, "ovs_interfaceid": "7cf93e54-e1d8-4b03-a06b-7164797d1bcc", "preserve_on_delete": false, "network": {"bridge": "br-int", "subnets": [{"ips": [{"meta": {}, "version": 4, "type": "fixed", "floating_ips": [], "address": "10.10.10.3"}], "version": 4, "meta": {}, "dns": [], "routes": [], "cidr": "10.10.10.0/24", "gateway": {"meta": {}, "version": 4, "type": "gateway", "address": "10.10.10.1"}}], "meta": {"injected": false, "tenant_id": "d4bbd972ca1b463ba028ccd1df983917", "mtu": 1500}, "id": "968d1118-8921-4ef9-9cf6-534ff3d8bf7d", "label": "net10"}, "devname": "tap7cf93e54-e1", "vnic_type": "normal", "qbh_params": null, "meta": {}, "details": {"port_filter": true, "ovs_hybrid_plug": true}, "address": "fa:16:3e:73:f8:2c", "active": true, "type": "ovs", "id": "7cf93e54-e1d8-4b03-a06b-7164797d1bcc", "qbg_params": null} unplug /usr/lib/python2.7/site-packages/nova/virt/libvirt/vif.py:966
2019-03-29 11:04:03.977 11524 DEBUG nova.network.os_vif_util [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Converting VIF {"profile": {}, "ovs_interfaceid": "7cf93e54-e1d8-4b03-a06b-7164797d1bcc", "preserve_on_delete": false, "network": {"bridge": "br-int", "subnets": [{"ips": [{"meta": {}, "version": 4, "type": "fixed", "floating_ips": [], "address": "10.10.10.3"}], "version": 4, "meta": {}, "dns": [], "routes": [], "cidr": "10.10.10.0/24", "gateway": {"meta": {}, "version": 4, "type": "gateway", "address": "10.10.10.1"}}], "meta": {"injected": false, "tenant_id": "d4bbd972ca1b463ba028ccd1df983917", "mtu": 1500}, "id": "968d1118-8921-4ef9-9cf6-534ff3d8bf7d", "label": "net10"}, "devname": "tap7cf93e54-e1", "vnic_type": "normal", "qbh_params": null, "meta": {}, "details": {"port_filter": true, "ovs_hybrid_plug": true}, "address": "fa:16:3e:73:f8:2c", "active": true, "type": "ovs", "id": "7cf93e54-e1d8-4b03-a06b-7164797d1bcc", "qbg_params": null} nova_to_osvif_vif /usr/lib/python2.7/site-packages/nova/network/os_vif_util.py:413
2019-03-29 11:04:03.978 11524 DEBUG nova.network.os_vif_util [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Converted object VIFBridge(active=True,address=fa:16:3e:73:f8:2c,bridge_name='qbr7cf93e54-e1',has_traffic_filtering=True,id=7cf93e54-e1d8-4b03-a06b-7164797d1bcc,network=Network(968d1118-8921-4ef9-9cf6-534ff3d8bf7d),plugin='ovs',port_profile=VIFPortProfileBase,preserve_on_delete=False,vif_name='tap7cf93e54-e1') nova_to_osvif_vif /usr/lib/python2.7/site-packages/nova/network/os_vif_util.py:425
2019-03-29 11:04:03.979 11524 DEBUG os_vif [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Unplugging vif VIFBridge(active=True,address=fa:16:3e:73:f8:2c,bridge_name='qbr7cf93e54-e1',has_traffic_filtering=True,id=7cf93e54-e1d8-4b03-a06b-7164797d1bcc,network=Network(968d1118-8921-4ef9-9cf6-534ff3d8bf7d),plugin='ovs',port_profile=VIFPortProfileBase,preserve_on_delete=False,vif_name='tap7cf93e54-e1') unplug /usr/lib/python2.7/site-packages/os_vif/__init__.py:112
2019-03-29 11:04:03.982 11524 DEBUG oslo.privsep.daemon [-] privsep: request[80071888]: (3, 'vif_plug_ovs.linux_net.delete_bridge', (u'qbr7cf93e54-e1', 'qvb7cf93e54-e1'), {}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:04:03.982 11524 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): brctl delif qbr7cf93e54-e1 qvb7cf93e54-e1 out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:04:03.995 11524 DEBUG oslo.privsep.daemon [-] CMD "brctl delif qbr7cf93e54-e1 qvb7cf93e54-e1" returned: 0 in 0.014s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:04:03.996 11524 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ip link set qbr7cf93e54-e1 down out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:04:04.004 11524 DEBUG oslo.privsep.daemon [-] CMD "ip link set qbr7cf93e54-e1 down" returned: 0 in 0.008s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:04:04.004 11524 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): brctl delbr qbr7cf93e54-e1 out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:04:04.021 11524 DEBUG oslo.privsep.daemon [-] CMD "brctl delbr qbr7cf93e54-e1" returned: 0 in 0.018s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:04:04.021 11524 DEBUG oslo.privsep.daemon [-] privsep: reply[80071888]: (4, None) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:04:04.023 11524 DEBUG oslo.privsep.daemon [-] privsep: request[80071888]: (3, 'vif_plug_ovs.linux_net.delete_ovs_vif_port', (u'br-int', 'qvo7cf93e54-e1'), {'timeout': 120}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:04:04.023 11524 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ovs-vsctl --timeout=120 -- --if-exists del-port br-int qvo7cf93e54-e1 out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:04:04.039 11524 DEBUG oslo.privsep.daemon [-] CMD "ovs-vsctl --timeout=120 -- --if-exists del-port br-int qvo7cf93e54-e1" returned: 0 in 0.016s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:04:04.039 11524 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ip link delete qvo7cf93e54-e1 out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:04:04.059 11524 DEBUG oslo.privsep.daemon [-] CMD "ip link delete qvo7cf93e54-e1" returned: 0 in 0.021s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:04:04.059 11524 DEBUG oslo.privsep.daemon [-] Net device removed: 'qvo7cf93e54-e1' out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:04:04.059 11524 DEBUG oslo.privsep.daemon [-] privsep: reply[80071888]: (4, None) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 11:04:04.060 11524 INFO os_vif [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Successfully unplugged vif VIFBridge(active=True,address=fa:16:3e:73:f8:2c,bridge_name='qbr7cf93e54-e1',has_traffic_filtering=True,id=7cf93e54-e1d8-4b03-a06b-7164797d1bcc,network=Network(968d1118-8921-4ef9-9cf6-534ff3d8bf7d),plugin='ovs',port_profile=VIFPortProfileBase,preserve_on_delete=False,vif_name='tap7cf93e54-e1')
2019-03-29 11:04:04.061 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 1462bd8c677c41b5a67b8c84c3e02be7 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:04.661 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 1462bd8c677c41b5a67b8c84c3e02be7 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:04.662 11524 DEBUG oslo_concurrency.lockutils [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lock "compute_resources" acquired by "nova.compute.resource_tracker.drop_move_claim" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2019-03-29 11:04:04.662 11524 DEBUG nova.objects.instance [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lazy-loading 'migration_context' on Instance uuid 4ad84d2b-babc-4ef6-87eb-437ba51a77dc obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1034
2019-03-29 11:04:04.663 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 909d3a23c4aa4d9888f885b499831dbd exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:04.672 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 909d3a23c4aa4d9888f885b499831dbd __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:04.678 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 5d148870c8644fe1bc6def012b62774a exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:04.718 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 5d148870c8644fe1bc6def012b62774a __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:04.719 11524 DEBUG nova.scheduler.client.report [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Refreshing aggregate associations for resource provider db732609-7867-486c-8e55-f52f70acfa62 _ensure_resource_provider /usr/lib/python2.7/site-packages/nova/scheduler/client/report.py:402
2019-03-29 11:04:04.752 11524 DEBUG oslo_concurrency.lockutils [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lock "compute_resources" released by "nova.compute.resource_tracker.drop_move_claim" :: held 0.090s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2019-03-29 11:04:04.755 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 72e062683aaa4df5bf8e75e796c61700 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:04.811 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 72e062683aaa4df5bf8e75e796c61700 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:04.814 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 21e81f70d46d4bea80ba898a7c56baec exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:05.295 11524 DEBUG oslo_service.periodic_task [req-d0c7f831-5df4-43f0-bede-60a00247d27e - - - - -] Running periodic task ComputeManager._instance_usage_audit run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 11:04:05.409 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 21e81f70d46d4bea80ba898a7c56baec __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:05.412 11524 WARNING nova.image.glance [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No protocol specified in for api_server 'http://172.30.126.56:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2019-03-29 11:04:05.415 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 89fc98cef9bb4efa84a7bdc4819fcb12 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:05.424 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 89fc98cef9bb4efa84a7bdc4819fcb12 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:05.424 11524 DEBUG oslo_concurrency.lockutils [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lock "4ad84d2b-babc-4ef6-87eb-437ba51a77dc" released by "nova.compute.manager.do_confirm_resize" :: held 5.412s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2019-03-29 11:04:05.426 11524 DEBUG oslo_messaging._drivers.amqpdriver [req-211524b7-bfaa-4074-b059-6439b555b6bc 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 09c95e94d0c04f35af277d2269c5c58f exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:05.482 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 09c95e94d0c04f35af277d2269c5c58f __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 11:04:09.054 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: ced488aae099430a94e3d3046a09aac8 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 11:04:10.390 11524 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: ced488aae099430a94e3d3046a09aac8 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407

